pkg/apis/postgres-operator.crunchydata.com/v1beta1/postgrescluster_types.go

```diff
@@ -66,6 +66,12 @@ type PostgresClusterSpec struct {
 	// +operator-sdk:csv:customresourcedefinitions:type=spec,order=1
 	Image string `json:"image,omitempty"`
 
+	// ImagePullPolicy is used to determine when Kubernetes will attempt to
+	// pull (download) PostgreSQL container images.
+	// https://kubernetes.io/docs/concepts/containers/images/#image-pull-policy
```

Comment:
⛏️ This affects more than the PostgreSQL containers.

```suggestion
	// pull (download) container images.
	// More info: https://kubernetes.io/docs/concepts/containers/images/#image-pull-policy
```